### PR TITLE
develop into master

### PR DESCRIPTION
**Pull request by Sergio1192**

### CODE REVIEW

File: src/Acheve.TestHost/Extensions/HttpResponseMessageExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+namespace System.Net.Http;
+
+public static class HttpResponseMessageExtensions
+{
+    /// <summary>
+    /// Try to extract the error message in the response content in case the response status code is not success.
+    /// </summary>
+    /// <param name="response">The HttpResponseMessage instance</param>
+    /// <returns></returns>
+    public static async Task IsSuccessStatusCodeOrThrow(this HttpResponseMessage response)
```
Comment (unaizorrilla) [inline src/Acheve.TestHost/Extensions/HttpResponseMessageExtensions.cs:13]: HttpResponseMessage already contains a similar method for this purpose ( i think without the content)

File: src/Acheve.TestHost/Extensions/TestServerExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    public static RequestBuilder CreateHttpApiRequest<TController, TActionResponse>(this TestServer server,
+        Expression<Func<TController, TActionResponse>> actionSelector,
+        object tokenValues = null,
+        RequestContentOptions contentOptions = null)
+        where TController : class
+        => UriDiscover.CreateHttpApiRequest<TController>(server, actionSelector, tokenValues, contentOptions);
+
+    public static IFormFile GivenFile(this TestServer _, string parameterName = "file", string filename = "test.txt", string content = "test")
+    {
+        var stream = new MemoryStream(Encoding.UTF8.GetBytes(content));
```
Comment (unaizorrilla) [inline src/Acheve.TestHost/Extensions/TestServerExtensions.cs:53]: is the stream disposed by formfile?

**Review by unaizorrilla — APPROVED** [review]
LGTM

File: src/Acheve.TestHost/Extensions/TestServerExtensions.cs (hunk 2 above)
Comment (Sergio1192) [reply src/Acheve.TestHost/Extensions/TestServerExtensions.cs:53]: Yes, in the api

**Pull request merged**